proto/autoops/service.proto

```diff
@@ -160,6 +161,67 @@ message ListWebhooksResponse {
   int64 total_count = 3;
 }
 
+message CreateProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  CreateProgressiveRolloutCommand command = 2;
+}
+
+message CreateProgressiveRolloutResponse {}
+
+message GetProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+}
+
+message GetProgressiveRolloutResponse {
+  ProgressiveRollout progressive_rollout = 1;
+}
+
+message DeleteProgressiveRolloutRequest {
+  string id = 1;
+  string environment_namespace = 2;
+  DeleteProgressiveRolloutCommand command = 3;
+}
+
+message DeleteProgressiveRolloutResponse {}
+
+message ListProgressiveRolloutRequest {
+  enum OrderBy {
+    DEFAULT = 0;
+    CREATED_AT = 1;
+    UPDATED_AT = 2;
+  }
+  enum OrderDirection {
+    ASC = 0;
+    DESC = 1;
+  }
+  string environment_namespace = 1;
+  int64 page_size = 2;
+  string cursor = 3;
+  repeated string feature_ids = 4;
+  OrderBy order_by = 5;
+  OrderDirection order_direction = 6;
+  optional ProgressiveRollout.Status status = 7;
+  string search_keyword = 8;
+  optional ProgressiveRollout.Type type = 9;
+}
+
+message ListProgressiveRolloutResponse {
+  repeated ProgressiveRollout progressive_rollouts = 1;
+  string cursor = 2;
+}
+
+message ExecuteProgressiveRolloutRequest {
+  string environment_namespace = 1;
+  string id = 2;
+  ChangeProgressiveRolloutTriggeredAtCommand
+      change_progressive_rollout_triggered_at_command = 3;
+}
+
+message ExecuteProgressiveRolloutResponse {
+  bool already_triggered = 1;
```

Comment:
Just confirming, are you going to use this flag?
I was checking the current code, and it only uses it for debugging.

https://github.com/bucketeer-io/bucketeer/blob/main/pkg/opsevent/batch/executor/executor.go#L74-L79

Removing it from the response will be better if you don't plan to use it.
If you want to use it, returning the `triggered_at` will be better.

Reply:
Giving it a second thought because this is different compared to the auto ops rules. If you need to return something, it should be the current status. What do you think?

Reply:
To keep the consistency (for debugging the same as auto ops) with auto ops rules, I added already_triggered flag.

Reply:
Let me remove it. Thank you.